src/renderer/__tests__/renderer.spec.tsx

```diff
@@ -129,14 +129,8 @@ describe('Renderer', () => {
       return null;
     }
 
-    let error = undefined;
-    try {
-      render(<Component />);
-    } catch(err) {
-      error = err;
-    }
-    // check substring of the desired error
-    expect((error as Error).message).toContain('Invalid hook call.');
+    // A console.error() will be printed due to illegal use of hooks but only in development mode
+    expect(() => render(<Component />)).toThrow("Cannot read properties of null (reading 'useState')");
```

Comment:
In `react@18` this no longer throws with `"Invalid hook call."`.
Instead it prints the error via `console.error()` but only in development mode